onedal/ensemble/forest.py

```diff
@@ -378,6 +387,7 @@ def _predict_proba(self, X, module, queue):
         return y
 
 
+@register_hyperparameters({"infer": get_hyperparameters("decision_forest", "infer")})
 class RandomForestClassifier(ClassifierMixin, BaseForest, metaclass=ABCMeta):
```

Comment:
Registration is attached in wrong place: it should be sklearnex, not onedal class.